java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/reading/BlobCheckpointStoreSample.java

```diff
@@ -0,0 +1,94 @@
+//package io.lenses.java.streamreactor.connect.azure.eventhubs.reading;
```

Comment:
why not delete this code?

Reply:
my bad, thanks for checking! deleted now